src/main/java/xogame/app/CurrentState.java

```diff
@@ -0,0 +1,19 @@
+package xogame.app;
+
+public class CurrentState implements GameState {
```

Comment:
CurrentState jest dość niefortunną nazwą, bo jak już nie będzie aktywny, to de facto nie będzie "Current" :) podejrzewam, że chodzi o stan w stylu "Game In Progress"